Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerTreeView.cpp

```diff
@@ -297,7 +300,7 @@ namespace AzToolsFramework
         QTreeView::timerEvent(event);
     }
 
-    void EntityOutlinerTreeView::processQueuedMousePressedEvent(QMouseEvent* event)
+    QMouseEvent EntityOutlinerTreeView::ToMousePressEvent(QMouseEvent* event)
```

Comment:
suggestion: Would could just make this a free function as we're not using any member date as far as I can tell?